src/plugins/vis_type_timelion/server/series_functions/es/index.js

```diff
@@ -96,20 +96,15 @@ export default new Datasource('es', {
       kibana: true,
       fit: 'nearest',
     });
-
-    const findResp = await tlConfig.savedObjectsClient.find({
```

Comment:
Since the `patternListActive` is generated at runtime, it is not stored on the `index-pattern` saved object. I changed this to fetch the index patterns from the `indexPatternsService.getPatternCache()`, which will give you a list of index patterns including the runtime `patternListActive` property